astropy/stats/lombscargle/implementations/scipy_impl.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function, division
+
+import numpy as np
+
+try:
```

Comment:
I'd move this inside `lombscargle_scipy`, so that `scipy` only gets imported when it is actually used. (I'd also change `ValueError` to `ImportError`, i.e., just adjust the error message)